plantuml2mysql.py

```diff
@@ -19,9 +18,8 @@ def strip_html_tags(t):
 
 # A minimal help
 def print_usage():
-  print("Convert PlantUML classes schema into mySQL database creation script")
+  print("Convert PlantUML classes schema into SQL database creation script")
```

Comment:
Well, I think the script promise compatibility for mysql only, so I reflected it in this message. I not planned it as a common script for many RDBMS though it could be applied with minor changes to other SQL-based systems.